src/kv/value.rs

```diff
@@ -615,6 +623,9 @@ mod std_support {
 
     impl<'v> Value<'v> {
         /// Try convert this value into a string.
+        // REVIEW: maybe rename this to contain COW somehow? I would expect &str
```

Comment:
Good point :+1: There's probably some precedence in the standard library to look at.

Reply:
I can't find much precedence in std lib. Using `rg --fixed-strings -e '-> Cow<'` all I could find was `IntoCow::into_cow` as a testing trait.

Maybe `to_cow_str`?

Reply:
`to_cow_str` sounds totally fine to me here.

Reply:
Created https://github.com/rust-lang/log/pull/592.